operations/template/alert.tf

```diff
@@ -0,0 +1,45 @@
+data "azurerm_monitor_action_group" "notify_slack_email" {
+  count               = local.non_pr_environment ? 1 : 0
+  resource_group_name = data.azurerm_resource_group.group.name
+  name                = "cdcti${var.environment}-actiongroup"
+}
+
+resource "azurerm_monitor_metric_alert" "azure_4XX_alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdc-rs-sftp-${var.environment}-azure-http-4XX-alert"
+  resource_group_name = data.azurerm_resource_group.group.name
+  scopes              = [azurerm_linux_web_app.sftp.id]
+  description         = "Action will be triggered when Http Status Code 4XX is greater than or equal to 3"
+  frequency           = "PT1M" // Checks every 1 minute
+  window_size         = "PT1H" // Every Check looks back 1 hour for 4xx errors
+
+  criteria {
+    metric_namespace = "Microsoft.Web/sites"
+    metric_name      = "Http4xx"
+    aggregation      = "Total"
```

Comment:
I had to change the aggregate type to **Total** (a.k.a. Sum) instead of **Count**. The reason why is because **Count** is the total number of measurements made within the granularity period. However, **Total** is the sum of values within that granularity period. For 4XX http calls, the **Total** is the total number of HTTP calls made in the granularity specified. While the Count is the total number of times Azure made a measurement within that granularity.

For Example: If Granularity is 1 minute, if I trigger **five** 4xx http calls within 1 minute, then the **Total** would be **five** but the count would only be **1** because it only measured it once.

Reply:
PR to update this in TI https://github.com/CDCgov/trusted-intermediary/pull/1486